homeassistant/components/nut/sensor.py

```diff
@@ -104,6 +107,7 @@ def __init__(
         self._unique_id = unique_id
 
         self._attr_name = f"{name} {sensor_description.name}"
+        self._attr_unique_id = f"{self._unique_id}_{self.entity_description.key}"
```

Comment:
```suggestion
        self._attr_unique_id = f"{unique_id}_{sensor_description.key}"
```

Can `unique_id` no longer be `None` ?

Reply:
thanks, I did miss the check